### ISSUE TITLE
Disable cardboard mode for iPads

### ISSUE THREAD

**Issue by dmarcos** [aframevr/aframe#4814]
There are no humongous cardboard holders as far as I know :)

We should perhaps offer a fullscreen button instead.

**Comment by vincentfretin** [comment]
With the changes in #5102
On iPad when you are in desktop mode (the default), the VR button make it fullscreen, but somewhat stretched horizontally. I'm not going to investigate that myself.
If you switch to mobile mode and click the VR button, you have the stereo working properly, but yeah not so useful with a tablet.

**Comment by dmarcos** [comment]
I'm committing some code to disable cardboard mode by default and start phasing it out. Cardboard holders are produced less and less and hard to come by the one that works for your device and screen size. The experience is not great, usage low and getting it right is becoming an intractable problem. The new behavior I'm aiming towards is:

- Desktop on a browser with WebXR. Show VR button
- Desktop on a browser without WebXR. Show fullscreen button (I'll implement this in a follow up commit)
- Mobile on a browser with WebXR support. Show VR / AR buttons when applicable.
- Mobile on a browser without WebXR support. Don't show any buttons.

**Comment by vincentfretin** [comment]
@dmarcos I don't know if you know it, your changes in commit https://github.com/aframevr/aframe/commit/042a3d6b7087a632c5165227b14bc37573375cde indeed disabled the VR button for Safari iPad but the button only disappear if you select mobile version of the page from the url bar. Desktop version is the default since years now on iPad, so we still can see the VR button by default here (and we get just a black render on desktop version). I'm not sure if we can better detect Safari iPad in desktop mode to hide the VR button. I don't have an iPhone, is the mobile version the default there? Does your changes hide the button on iPhone?
On Chome Android, we still see the VR button.

**Comment by dmarcos** [comment]
On Chrome Android WebXR is available hence the VR button is displayed. On iPad I will test but VR button should be expected until I replace the icon with a fullscreen one (same as regular desktop). Behavior should be entering fullscreen but not sure what's the expected Safari for iPad fullscreen situation anymore. There should not be any buttons displayed on iPhone

**Comment by vincentfretin** [comment]
Ok, all good then for Chrome Android.
For iPhone, Cardboard is working again since we fixed WebVRManager in #5102 but it may break again in the future if nobody is actively testing it and fixing it that's for sure so I understand we want to disable it by default.
For Safari iPad, last time I checked, it didn't have the fullscreen api. The only way to have the page fullscreen is to add it to your home screen first. I'd say we shouldn't have the VR or fullscreen button at all for Safari iPad in desktop mode and neither in mobile mode. On desktop mode the user agent is the same as Safari macOS if I remember, so it may be difficult to detect that this is an iPad, but I don't know for sure.
